cmd/gokys/main.go

```diff
@@ -52,16 +52,30 @@ func main() {
 	cfg := readCfg()
 	files := getFiles(flag.Args())
 
-	fset := token.NewFileSet()
-	measurer := wmfp.NewMeasurerWMFP(&cfg)
-
+	measurers := make(chan *wmfp.MeasurerWMFP, len(files))
 	for _, file := range files {
-		node, err := parser.ParseFile(fset, file, nil, parser.ParseComments)
-		die(err)
-		measurer.ParseFile(node)
+		go measureFile(file, cfg, measurers)
 	}
 
-	fmt.Println(measurer.Finish())
+	fmt.Println(combineMeasures(measurers, len(files)))
+}
+
+func measureFile(file string, config wmfp.Config, results chan *wmfp.MeasurerWMFP) {
+	measurer := wmfp.NewMeasurerWMFP(&config)
+	node, err := parser.ParseFile(token.NewFileSet(), file, nil, parser.ParseComments)
+	die(err)
```

Comment:
Bad idea. If one of the files doesn't exist, then it will just be a deadlock.

Reply:
So go will panic, just like it did before, right?

Reply:
Hmm, makes sence then